task_processing/plugins/mesos/execution_framework.py

```diff
@@ -199,6 +199,33 @@ def _background_check(self):
             )
             time.sleep(10)
 
+    def reconcile_task(self, task_config):
```

Comment:
mesos supports 'implicit' reconciliation, when you can call `reconcile` with an empty list of task ids and it returns everything. do we want to support that?

Reply:
Sure, we could let `task_config` be optional in all the `reconcile` methods and do implicit reconciliation in the framework if the config is None. 

That kind of makes the `reconcile` method more specific to Mesos, but if we think it would be useful to have implicit reconciliation, I don't think that matters too much.

Reply:
Just remembered the framework currently discards status updates for tasks that don't exist in task_metadata, so we'd have to decide how to deal with that, too. Since I wasn't planning on using implicit reconciliation in Tron for now, how about leaving this until we decide we need it?